src/iop/basecurve.c

```diff
@@ -1783,6 +1828,15 @@ static void exstep_callback(GtkWidget *widget, gpointer user_data)
   dt_dev_add_history_item(darktable.develop, self, TRUE);
 }
 
+static void exbias_callback(GtkWidget *widget, gpointer user_data)
```

Comment:
I followed the same pattern as elsewhere in the same file: name the callback after the name of the widget. Best is probably to rename both widgets and callbacks: I've just done that it 0885e55. Indeed I like the code better after than before: the same naming convention is used for internal data and for widgets now.